docs/intro/glossary.md

```diff
@@ -0,0 +1,255 @@
+# Glossary
+
+This glossary provides definitions for common terminology used in the SDK,
+focusing on microgrid components, metrics, measurements, and power-related
+terms.
+
+## Common Acronyms
+
+### AC
+
+Alternating current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Alternating_current) for more details.
+
+### BMS
+
+Battery management system. See the [Wikipedia article](https://en.wikipedia.org/wiki/Battery_management_system) for more details.
+
+### CHP
+
+Combined heat and power. See the [Wikipedia article](https://en.wikipedia.org/wiki/Combined_heat_and_power) for more details.
+
+### DC
+
+Direct current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Direct_current) for more details.
+
+### EV
+
+Electric vehicle. See the [Wikipedia article](https://en.wikipedia.org/wiki/Electric_vehicle) for more details.
+
+### PV
+
+Photovoltaic. See the [Wikipedia article](https://en.wikipedia.org/wiki/Photovoltaics) for more details.
+
+In the SDK it is normally used as a synonym for [solar panel](#solar-panel).
+
+### SoC
+
+[State of charge](#state-of-charge). See the [Wikipedia article](https://en.wikipedia.org/wiki/State_of_charge) for more details.
+
+### SoP
+
+[State of power](#state-of-power).
+
+## Microgrid
+
+A local electrical grid that connects a set of different [types of components](#component-category) together. It can be connected to the main grid, or be completely isolated.
+
+### Component Category
+
+The different types of [components](#component) a [microgrid](#microgrid) can have, such as an inverter, battery, meter, etc.
+
+Devices of the same category have the same characteristics (for example offer the same set of [metrics](#metric)).
+
+#### Battery
+
+A storage system for electrical energy.
+
+#### CHP Plant
+
+A generator that produces combined heat and power ([CHP](#chp)). Usually powered via combustion of some form of fuel.
+
+#### Converter
+
+A [DC](#dc)-DC converter.
+
+#### EV Charger
+
+A station for charging [EVs](#ev).
+
+#### Electrolizer
+
+An device that converts water into hydrogen and oxygen.
+
+#### Grid
+
+A point where the local [microgrid](#microgrid) is connected to the main electricity grid.
```

Comment:
This was also taken from the common API docs. I now replaced `main` with `public`, do you think that's enough or do you think it needs further clarification?

Reply:
I gave it a bit of thought.
When we say `Grid`, we mean an abstraction over the whole grid, encapsulated into a single component in the microgrid. We do not actually mean the grid connection point anymore. This could certainly have effect on the documentation on our APIs too.

I think it's fine to leave it like this for now. We can discuss the terminology in a separate issue/discussion.